addons/sale/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,197 @@
+---Fields in module 'sale'---
+sale         / account.analytic.line    / so_line (many2one)            : NEW relation: sale.order.line
```

Comment:
it might make sense to guess best matches, we should come quite far when matching the line's name and/or the product involved